manage.py

```diff
@@ -1,37 +1,56 @@
 #!/usr/bin/env python
 import os
 import sys
+from coverage import Coverage
```

Comment:
Coverage module might not be available when running in production as we are not installing dependencies in requirements.dev.txt by default. Please move import into `run_tests_with_coverage` function.